example/zcat/README.md

```diff
@@ -1,36 +1,37 @@
 # Overview
 This example illustrates how to embed zero trust connectivity into a Netcat like application. There are two parts, part
- 1 shows how to use zcat with a broadcast style messaging service. Part two shows how to use addressable terminators to
- direct messages to specific endpoints.
+ 1 shows how to use zcat with a broadcast style messaging service. Part two shows how to use 
+ [addressable terminators](https://github.com/openziti/fabric/wiki/Addressable-Terminators) to direct messages to 
+ specific endpoints.
 
 This example demonstrates:
 * Dialing a service
 * Binding a service
-* Using addressable terminators
+* Using [addressable terminators](https://github.com/openziti/fabric/wiki/Addressable-Terminators)
```

Comment:
:/ can you put an issue into ziti-doc to copy/move/publish this on the docs site, then come back here and update this link once published?

Reply:
Done https://github.com/openziti/ziti-doc/issues/214